entity-framework/core/extensions/index.md

```diff
@@ -342,6 +342,14 @@ For EF Core: 6, 7.
 
 [GitHub repository](https://github.com/R4ND3LL/EntityFrameworkRuler/) | [CLI Tool NuGet](https://www.nuget.org/packages/EntityFrameworkRuler/) | [Design NuGet](https://www.nuget.org/packages/EntityFrameworkRuler.Design/)
 
+### Moq.EntityFrameworkCore
+
+This library helps you mocking EntityFramework contexts. Now you will be able to test methods that are using `DbSet<TEntity>` or `DbQuery<TEntity>` from `DbContext` in an effective way.
+
+For EF Core: 3, 5, 6, 7.
```

Comment:
Please move up to the paragraph above - otherwise LGTM

Reply:
I moved to the paragraph above (Tools). I doubted whether this should be seen as a Tool or an Extension.